src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesignerOptionService.cs

```diff
@@ -33,7 +33,7 @@ public DesignerOptionCollection Options
         /// <summary>
         ///     Creates a new DesignerOptionCollection with the given name, and adds it to 
         ///     the given parent.  The "value" parameter specifies an object whose public 
-        ///     properties will be used in the Propeties collection of the option collection.  
+        ///     properties will be used in the Properties collection of the option collection.  !!maybe small properties here?
```

Comment:
Nope, capital is correct. See the `Properties` property (which is of a collection type) below.